features/old/api/features.feature

```diff
@@ -8,59 +8,63 @@ Feature: Features :)
     And an application plan "Basic" of provider "foo.3scale.localhost"
     Given current domain is the admin domain of provider "foo.3scale.localhost"
 
-  @javascript @wip
+  @javascript
   Scenario: Create a feature
     When I log in as provider "foo.3scale.localhost"
-    And I go to the edit page for plan "Basic"
+    Then I go to the edit page for plan "Basic"
     And I follow "New feature"
     And I fill in "Name" with "Free T-shirt"
+    And I fill in "System name" with "tee"
     And I fill in "Description" with "T-shirt with logo of our company for free."
-    And I press "Create Feature"
-    Then I should see enabled feature "Free T-shirt"
-    And provider "foo.3scale.localhost" should have feature "Free T-shirt"
-    And feature "Free T-shirt" should be enabled for plan "Basic"
+    And I press "Save"
+    Then I should see "T-shirt with logo of our company for free."
 
   # TODO: Test that the "No features yet" notice dissapears when first feature is created,
   #       and appears when last one is deleted.
-
-  @javascript @wip
+  @javascript @chrome
```

Comment:
We can use `@chrome` for local testing, but shouldn't commit it.

Reply:
yes I pushed it by mistake not intentionally